### PR TITLE
Upgrade BBB to v2.3, update doc, add testing inventory

### PR DESCRIPTION
**Pull request by anadahz**
- Update documentation of vault usage examples
- Add testing inventory
- Bump docker role version to version 3.1.2
- Bump BBB role to git commit 7900bc196a028e2f899b47d7a0461573e439f58b
- Add default presentation PDF file
- Login to docker to deal with rate limits in CI
- Change molecule test target to Ubuntu 18.04
- Generate BBB secrets if not defined in the inventory
- Add custom PDF file of the presentation
- Print output of `bbb-conf --check`

Fixes:

- https://github.com/PrivacyLx/privacylx-issue-tracker/issues/98
- https://github.com/PrivacyLx/privacylx-issue-tracker/issues/97
- https://github.com/PrivacyLx/privacylx-issue-tracker/issues/93

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,16 +6,16 @@ python: "3.8"
...
+#  - name: "Testing ansible playbook `deploy-update.yml`"
+#    env: TEST_ROLE=update
+#  - name: "Testing ansible playbook `deploy-discourse.yml`"
+#    env: TEST_ROLE=discourse
   - name: "Testing ansible playbook `deploy-bigbluebutton.yml`"
     env: TEST_ROLE=bigbluebutton
-  - name: "Testing ansible playbook `deploy-matterbridge.yml`"
-    env: TEST_ROLE=matterbridge
+#  - name: "Testing ansible playbook `deploy-matterbridge.yml`"
+#    env: TEST_ROLE=matterbridge
```
Comment (anadahz) [inline .travis.yml:18]: TODO: Enabled when ready

File: ansible/roles/bigbluebutton/defaults/main.yml [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-# Git commit or release version of bbb_role_git
-bbb_role_ver: 0104e9eefcaf88b9129a287953984fcaa63adc41
-# https://github.com/n0emis/ansible-role-bigbluebutton
-bbb_galaxy_role: n0emis.bigbluebutton
+# https://github.com/ebbba-org/ansible-role-bigbluebutton/releases
+bbb_role_ver: 609713bb5fccf9910b0a3545e789a8d75c62f9db
+bbb_galaxy_role: ebbba.bigbluebutton
+bbb_git_role: https://github.com/ebbba-org/ansible-role-bigbluebutton
+# Default presentation PDF file located in the files role directory
+bbb_default_pdf: default.pdf
```
Comment (francisco-core) [inline ansible/roles/bigbluebutton/defaults/main.yml:9]: We have here the default presentation with a privacylx logo. If someone wants to / is reusing our role, they need to change this. I was trying to look at a way to store this "default.pdf" somehow in the inventory, but I didn't find a way. So I guess for the time being this is fine.

**Review by francisco-core — APPROVED** [review]
Other than that it all seems good

File: ansible/roles/bigbluebutton/defaults/main.yml (hunk 2 above)
Comment (anadahz) [reply ansible/roles/bigbluebutton/defaults/main.yml:9]: Thanks for the review @francisco-core .
If someone would like to change the default pdf, they can change this variable in their inventory or during the run of the playbook.

**Comment by anadahz** [comment]
BTW the network failure is something related to Travis CI, for reference:
https://travis-ci.community/t/network-issues-temporary-failure-in-name-resolution/11622/

**Pull request merged**